site/src/components/pages/docs/search/search-list.tsx

```diff
@@ -1,23 +1,27 @@
+/* eslint-disable no-param-reassign */
 import { faBookOpen } from "@fortawesome/free-solid-svg-icons";
-import { Box, Divider, useTheme } from "@mui/material";
-import React, { useMemo } from "react";
+import { Box, Collapse, Divider, useTheme } from "@mui/material";
+import React, { FC, useMemo } from "react";
+import { TransitionGroup } from "react-transition-group";
 
 import { FontAwesomeIcon, SpecificationIcon } from "../../../icons";
 import { AlgoliaHighlightResult, AlgoliaResult, SearchVariants } from "./index";
 import SearchListCategory from "./search-list-category";
 
-interface SearchListProps {
+type SearchListProps = {
   searchResults: AlgoliaResult[];
+  searchListItemsRefs: React.MutableRefObject<HTMLButtonElement[]>;
   variant?: SearchVariants;
   getHighlight: (highlight: AlgoliaHighlightResult) => string;
   closeModal?: () => void;
-}
+};
 
-const SearchList: React.VoidFunctionComponent<SearchListProps> = ({
+const SearchList: FC<SearchListProps> = ({
```

Comment:
Right now the maximum number of results is 10. I'm not even sure there's a query that manages 10 hits, but if there is one the modal will stretch outside the window view. Are you suggesting reducing the maximum number of hits, or letting the modal stretch and move the scrollbar outside of the modal?

Reply:
I was suggesting removing the inner scroll-bar that's visible within the screenshot, but am realising I hadn't considered what would happen when the modal exceeds the screen height where a scrollbar may make sense.

Could you set a max-height to the modal that's relative to the screen height and keep the scrollbar, so that it doesn't overflow like this? 

https://user-images.githubusercontent.com/42802102/175566484-c95c8307-352c-4a33-b0af-74cb27ab9a99.mov

Reply:
Addressed in a4d90c62407628be111c65ffdb22de5c57895216

Reply:
Perfect, thank you 👍